src/libraries/Microsoft.PowerFx.Core/Binding/BindInfo/FirstNameInfo.cs

```diff
@@ -18,6 +18,10 @@ internal sealed class FirstNameInfo : NameInfo
     {
         public override DName Name => Node.AsFirstName().Ident.Name;
 
+        // DisplayName is set in Create methods
+        // Mostly used with identifiers in PreVisit(CallNode node)
+        public DName DisplayName { get; init; }
```

Comment:
Does anybody actually use this? 
Binder is usually translating to *logical* names  and away from DisplayNames.

Reply:
You're correct, this isn't necessary anymore